### ISSUE TITLE
submit does not work in runcpm for arduino due

### ISSUE THREAD

**Issue by gfernval** [MockbaTheBorg/RunCPM#55]
Submit.com and submitd.com doesn´t appear to work. The only program that appears to work is supersub.com, a more powerful submit-like utility, but only works in A: drive, not B:, C:,..

**Comment by gfernval** [comment]
You are OK, when editig .SUB files with TE editor care must be taken of NOT putting a blank empty line at the end. In past issues one of your messages states: "Found the issue ... SUBMIT has a bug ... if the last line of the .SUB is empty, it bails out without writing anything.
If you leave empty lines on the script, it also fails.
Same happens on a real CP/M machine."

**Comment by gfernval** [comment]
Hello:
You are OK, when editig .SUB files with TE editor care must be taken of NOT putting a blank empty line at the end. In past issues one of your messages states: "Found the issue ... SUBMIT has a bug ... if the last line of the .SUB is empty, it bails out without writing anything.
If you leave empty lines on the script, it also fails.
Same happens on a real CP/M machine."
thanks: gonzalo


________________________________
De: Mockba the Borg <notifications@github.com>
Enviado: viernes, 15 de junio de 2018 14:26
Para: MockbaTheBorg/RunCPM
Cc: gfernval; Author
Asunto: Re: [MockbaTheBorg/RunCPM] submit does not work in runcpm for arduino due (#55)


Hi there ... thanks for reporting the issue here. This helps our small community to see all the problems and assist as well.

I just executed a test here:
Using my Arduino I went to drive C and created a TEST.SUB file there, with the following contents:

dir
a:stat

When I created the file, I accidentally left an extra empty line at the end, meaning, there's a blank like below "a:stat".
With that line there, when I do A:SUBMIT TEST nothing happens. If I delete the blank line the submit works fine.
Looks like blank lines on the .SUB file are causing the problems.

I still need to determine if this is a RunCPM issue or a CP/M issue itself.

Is this the issue that you are experiencing?
